app/controllers/application_controller.rb

```diff
@@ -74,6 +73,7 @@ def authenticate_user_with_basic_auth!
   def authenticate_user!
     type, credentials = type_and_credentials_from_request_headers
     return false if credentials.blank?
+    raise JWT::VerificationError if (ENV['JWT_BLACKLISTED'] == credentials)
```

Comment:
I would make  `JWT_BLACKLISTED` a comma-separated string that is converted to a list. There is no reason to assume there will only be one blacklisted JWT.

Reply:
i was hoping it would be a one-off, but you are right. I'll change that